mkdocs.yml

```diff
@@ -135,7 +135,7 @@ nav:
       - Installing the Configurator: quick-start/installing-configurator.md
       - Firmware Options: quick-start/firmware-options.md
       - Transmitters:
-          - Preparing your Radio: quick-start/transmitters/tx-prep.md
```

Comment:
I'm not very sure about the removal of the word.

If it's going to be changed, might as well use "Radio Preparation"